### ISSUE TITLE
Galleries do not have image height and width which results in a weird layout

### ISSUE THREAD

**Issue by HexeBianca** [IMAGINARY/imaginary-web#27]
Below a gallery 'more galleries' are shown in a weird spacing.
Example:
http://imaginary.org/gallery/quasicrystalline-wickerwork
![1](https://cloud.githubusercontent.com/assets/10153723/5543641/774ed68c-8af7-11e4-81f9-2ff2c5fa0fa0.jpg)

**Comment by christophknoth** [comment]
I found out that without the autorotate function in the image style this bug does not happen. So I removed it for now in the image style 290. But we should update the related module image cache actions and see if the bug still persists. Somehow the bug does not happen on my dev server.

One thing we should discuss if wie do an update if it would be necessary to still have a gallerie-function (show all images already) for the related galleries. It is a bit redundant somehow.
